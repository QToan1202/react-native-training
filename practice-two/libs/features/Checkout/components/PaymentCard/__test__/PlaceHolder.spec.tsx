import { render } from '@testing-library/react-native'

import PaymentCardPlaceHolder from '../PlaceHolder'

describe('Testing Payment Card placeholder component', () => {
  it('Render Payment Card placeholder component success', () => {
    const component = render(<PaymentCardPlaceHolder />).toJSON()

    expect(component).toMatchSnapshot()
  })
})
