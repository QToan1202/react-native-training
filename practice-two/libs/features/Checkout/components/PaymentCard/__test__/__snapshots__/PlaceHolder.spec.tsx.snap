// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Payment Card placeholder component Render Payment Card placeholder component success 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    [
      {
        "alignItems": "center",
        "alignSelf": "flex-start",
        "borderColor": "#00000033",
        "borderRadius": 10,
        "borderStyle": "dashed",
        "borderWidth": 2,
        "paddingHorizontal": 38.5,
        "paddingVertical": 36,
        "rowGap": 20,
      },
      undefined,
    ]
  }
>
  <Image
    source={
      {
        "testUri": "../../../assets/payment/icon.png",
      }
    }
  />
  <Text
    style={
      [
        {
          "color": "#fff",
          "fontFamily": "Montserrat_400Regular",
        },
        {
          "fontSize": 14,
        },
        {
          "color": "#00000033",
          "letterSpacing": 0.5,
          "lineHeight": 28,
          "textTransform": "capitalize",
        },
      ]
    }
  >
    add payment method
  </Text>
</View>
`;
